src/server/lib/spreadsheet.js

```diff
@@ -79,12 +80,20 @@ function parseSpreadsheet (workbook, templateSheets) {
   })
 
   // convert the sheets from xlsx format to AOA format
-  const parsedWorkbook = _.mapValues(
-    normalizedSheets || {},
-    sheet => {
-      return sheetToJson(sheet)
-    }
-  )
+  const jsonWorkbook = _.mapValues(normalizedSheets || {}, sheet => {
+    return sheetToJson(sheet)
+  })
+
+  // remove metadata rows & normalize column names
+  const parsedWorkbook = {}
+  Object.entries(jsonWorkbook).forEach(([sheetName, sheet]) => {
+    const normalizedSheet = removeMetadata(sheetName, jsonWorkbook[sheetName])
+    const [headerRow, ...rest] = normalizedSheet
+    const normalizedHeader = _.map(headerRow, colName =>
+      typeof colName === 'string' ? colName.toLowerCase().trim() : colName
+    )
+    parsedWorkbook[sheetName] = [normalizedHeader, ...rest]
+  })
```

Comment:
this is a great candidate for a unit test; it could be broken out into a separate function for ease of testing